### DISCUSSION TITLE
Disable sensitive content warning when logged in?

### DISCUSSION THREAD

**Discussion by flypenguin** [pixelfed/pixelfed#5065] [Q&A]
Hello Pixelfed community, I just saw a some "sensitive content" images while getting to know my brand new Pixelfed account :) . Unfortunately I am very annoyed by those warnings, so I was wondering if I could disable them while being logged in to my account?

Thanks in advance!

**Comment by keithfancher** [comment]
I definitely think this should be an option as well.

For starters it could be global, but it would also be a cool addition, I think, to be able to whitelist particular users or servers, etc. Maybe limit it to only people you follow, etc.

Also, it would be nice if you could "click to show" in the search view without having to actually *leave* the search view, load the post, and then click *again* to show the actual contents. Or even just a toggle for that search so you can disable the filtering altogether in that particular search.

**Comment by flypenguin** [comment]
Is it _really_ only the two of us who would like that?

**Comment by tsargoth** [comment]
I'd love this feature as well

**Comment by codeinabox** [comment]
The Android app Pixelix has implemented this feature https://github.com/daniebeler/pixelix/issues/271
